scripts/metricsdocgen/metrics

```diff
@@ -0,0 +1,576 @@
+# HELP coderd_api_active_users_duration_hour The number of users that have been active within the last hour.
+# TYPE coderd_api_active_users_duration_hour gauge
+coderd_api_active_users_duration_hour 0
+# HELP coderd_api_concurrent_requests The number of concurrent API requests
+# TYPE coderd_api_concurrent_requests gauge
+coderd_api_concurrent_requests 1
+# HELP coderd_api_concurrent_websockets The total number of concurrent API websockets
+# TYPE coderd_api_concurrent_websockets gauge
+coderd_api_concurrent_websockets 1
+# HELP coderd_api_request_latencies_ms Latency distribution of requests in milliseconds
+# TYPE coderd_api_request_latencies_ms histogram
+coderd_api_request_latencies_ms_bucket{method="GET",path="",le="1"} 10
```

Comment:
Wonder if we could get a cleaner dump here, the histogram seems to take up a lot of space, but it's just repeating different routes.

Reply:
Frankly speaking, the idea was to just dump it and forget about it. The format is not intended to be reviewed by humans. The generator makes sure that it can be cleaned and processed into the Markdown table.

Regarding the dumping process, I considered other options like booting up the coderd and fetching Prometheus metrics from http://localhost:2112, but you would have to run it in a container as process metrics are only available in Linux. Eventually, I decided to leave it to the developer to just use `curl` and update the file.

If you have any ideas on how to improve it, I'm happy to change the concept.

Reply:
What if instead of storing the dump in the repo you made the script directly connect to the server and download the prom metrics? Would probably need to be moved out of `make gen` as it would be prone to failure if not configured properly.